poll/poll.go

```diff
@@ -0,0 +1,140 @@
+package poll
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+)
+
+const (
+	optionPrefix    = "-"
+	lineBreakSuffix = "\n"
+)
+
+var DefaultConfig = PollConfig{
```

Comment:
Document ALL exported types and methods please. But also make sure everything you export needs to be exported